db/migrate/20240503213904_create_units_trajectories_models.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUnitsTrajectoriesModels < ActiveRecord::Migration[7.1]
+  def change
+    create_table :units_trajectories_models do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
❓

Reply:
same question that @mtopete1996 , what are you trying to do here?